ONSdigital/census-rm-action-service
# Census address format

# Motivation and Context
The address format is changing for census attributes, including an address line 3. This PR brings the [actionsvc-api](https://github.com/ONSdigital/rm-actionsvc-api) classes into this project where they were previously imported as a dependency. This allows us to change them without worrying about setting up a separate census version building to an artifactory (though we will have to make sure it remains identical accross the other projects which use then transport classes). Also renames some social specific code to census where it has now been repurposed.

# What has changed
- Bring in actionsvc-api transport classes, xsd's and tests
- Add pom goals to generate sources from xsd's
- Add address line 3 to the action address xsd
- Add test for sample attributes request decorator
- Rename repurposed social specific code to census and update tests accordingly 

# How to test?
This is waiting on changes to the casesvc, actionexportersvc and sample loader before it can easily be manually tested. For now you can run the integration tests and make sure they make sense.

# Links
https://trello.com/c/e9iWXgje/517-rmc-83-load-hh-sample-and-generate-print-file-to-census-spec-13
